site/content/en/docs/tasks/run/topology_aware_scheduling.md

```
@@ -214,6 +214,55 @@ kubectl -n default get workloads.kueue.x-k8s.io <workload-name> -o yaml
+#### Verify rack co-location
+
+After the LeaderWorkerSet is admitted, list all Pods and the node they were assigned to, along with the rack label of that node:
+
+```shell
+kubectl get pods -l leaderworkerset.sigs.k8s.io/name=nginx-leaderworkerset \
+  -o custom-columns='NAME:.metadata.name,NODE:.spec.nodeName,GROUP:.metadata.labels.leaderworkerset\.sigs\.k8s\.io/group-index,WORKER:.metadata.labels.leaderworkerset\.sigs\.k8s\.io/worker-index'
+```
+
+The output is similar to the following:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)